ui/src/app/cohort-search/list-overview/list-overview.component.tsx

```diff
@@ -0,0 +1,426 @@
+              }
+              {this.hasErrors && <span>
+                -- <TooltipTrigger content={this.hasTemporalError ?
+                    'Please complete criteria selections before saving temporal relationship.' :
+                    `All criteria are suppressed. Un-suppress criteria to update the total count
+                     based on the visible criteria.`}>
+                  <ClrIcon style={{color: '#F57600'}} shape='warning-standard' size={18} />
+                </TooltipTrigger>
+              </span>}
+            </h2>
+          </div>
+          {error && !this.hasErrors && <div style={styles.totalError}>
+            <ClrIcon className='is-solid' shape='exclamation-triangle' size={22} />
+            Sorry, the request cannot be completed.
+          </div>}
+          {!this.hasErrors && !loading && !!chartData && total && <div style={styles.cardContainer}>
+            <div style={styles.card}>
+              <div style={styles.cardHeader}>
+                Results by Gender
+              </div>
+              <div style={{padding: '0.5rem 0.75rem'}}>
+                {chartData.size && <GenderChart data={chartData} />}
+              </div>
+            </div>
+            <div style={styles.card}>
+              <div style={styles.cardHeader}>
+                Results By Gender, Age Range, and Race
+                <ClrIcon shape='sort-by'
+                  className={stackChart ? 'is-info' : ''}
+                  onClick={() => this.toggleChartMode()} />
+              </div>
+              <div style={{padding: '0.5rem 0.75rem'}}>
+                {chartData.size &&
+                  <ComboChart mode={stackChart ? 'stacked' : 'normalized'} data={chartData} />}
+              </div>
+            </div>
+          </div>}
+        </div>
+        {saveModal && <Modal>
+          <ModalTitle style={invalid ? {marginBottom: 0} : {}}>Save Cohort as</ModalTitle>
+          <ModalBody style={{marginTop: '0.2rem'}}>
+            {saveError && <div style={styles.error}>
+              <ClrIcon className='is-solid' shape='exclamation-triangle' size={22} />
+              Data cannot be saved. Please try again.
+            </div>}
+            {invalid && <div style={styles.invalid}>Cohort name is required</div>}
+            <TextInput style={{marginBottom: '0.5rem'}} value={name} placeholder='COHORT NAME'
+              onChange={(v) => this.setState({name: v, nameTouched: true})}
+              disabled={saving} />
+            <TextArea value={description} placeholder='DESCRIPTION' disabled={saving}
+              onChange={(v) => this.setState({description: v})}/>
+          </ModalBody>
+          <ModalFooter>
+            <Button style={{color: '#262262'}} type='link' onClick={() => this.setState({
+              saveModal: false, name: undefined, description: undefined, saveError: false,
+              nameTouched: false
+            })} disabled={saving}>CANCEL</Button>
```

Comment:
nit: button styling automatically capitalizes; this doesn't need to be capitalized